ui/hooks/ramps/useRamps/useRamps.test.tsx

```diff
@@ -124,22 +127,39 @@ describe('useRamps', () => {
     jest.resetModules();
   });
 
-  it('should handle Solana chain IDs correctly', () => {
+  it('should handle EVM chain IDs correctly by converting to numeric format', () => {
     const metaMaskEntry = 'ext_buy_sell_button';
-    const solanaChainId = 'solana:4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ';
-    (isSolanaChainId as jest.Mock).mockReturnValueOnce(true);
+    const evmChainId = '0x1';
+    (isEvmChainId as jest.Mock).mockReturnValueOnce(true);
+    mockStoreState = {
+      ...mockStoreState,
+      metamask: {
+        ...mockStoreState.metamask,
+        ...mockNetworkState({ chainId: evmChainId }),
+      },
+    };
+    const mockBuyURI = `${process.env.PORTFOLIO_URL}/buy?metamaskEntry=${metaMaskEntry}&chainId=1&metametricsId=${mockedMetametricsId}&metricsEnabled=false`;
+    const { result } = renderHook(() => useRamps(), { wrapper });
+    const buyURI = result.current.getBuyURI(evmChainId);
+    expect(buyURI).toBe(mockBuyURI);
+  });
+
+  it('should handle non-EVM chain IDs correctly (like Bitcoin)', () => {
+    const metaMaskEntry = 'ext_buy_sell_button';
+    const bitcoinChainId = 'bitcoin:mainnet';
```

Comment:
is `bitcoin:mainnet` a realistic value?on the previous test  it was `solana:4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ`, can you use `bip122:000000000019d6689c085ae165831e93`? is that what extension is using right?